tests/python/test_tools_memleak.py

```diff
@@ -102,7 +102,7 @@ def test_memalign(self):
         self.assertEqual(cfg.leaking_amount, self.run_leaker("memalign"))
 
     def test_pvalloc(self):
-        self.assertEqual(cfg.leaking_amount, self.run_leaker("pvalloc"))
+        self.assertLessEqual(cfg.leaking_amount, self.run_leaker("pvalloc"))
```

Comment:
There is a [valloc](https://linux.die.net/man/3/pvalloc) test above, does it fail ?

Reply:
No. Its only observed for pvalloc.